<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUpdateExtensionsBaseDialog</class>
 <widget class="QDialog" name="CUpdateExtensionsBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Extensions</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="groupBox16">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="m_textextensionsnotfound">
        <property name="text">
         <string>The following extensions have been found, 
but are not defined in any of the extension lists:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Q3ListBox" name="m_listboxextensionsnotfound">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
        <property name="selectionMode">
         <enum>Q3ListBox::Single</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_textwhattodo">
        <property name="text">
         <string>Which of the following actions should be taken?</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkboxdownloadheaders">
        <property name="text">
         <string>Download the latest extension header files</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkboxreadheaders">
        <property name="text">
         <string>Read the extension header files</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkboxdownloadregistry">
        <property name="text">
         <string>Download the latest registry specification files</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkboxreadregistry">
        <property name="text">
         <string>Read the registry specification files</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_checkboxignoreextensions">
        <property name="text">
         <string>Add these extensions to the ignored list</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pushbuttonreset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>240</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttoncancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonapply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_checkboxreadheaders</sender>
   <signal>toggled(bool)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>checkboxreadheaders_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkboxdownloadheaders</sender>
   <signal>toggled(bool)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>checkboxdownloadheaders_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkboxreadregistry</sender>
   <signal>toggled(bool)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>checkboxreadregistry_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkboxdownloadregistry</sender>
   <signal>toggled(bool)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>checkboxdownloadregistry_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkboxignoreextensions</sender>
   <signal>toggled(bool)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>checkboxignoreextensions_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_listboxextensionsnotfound</sender>
   <signal>doubleClicked(Q3ListBoxItem*)</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>listboxextensionsnotfound_doubleClicked(Q3ListBoxItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonreset</sender>
   <signal>clicked()</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>pushbuttonreset_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttoncancel</sender>
   <signal>clicked()</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>pushbuttoncancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonapply</sender>
   <signal>clicked()</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>pushbuttonapply_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonok</sender>
   <signal>clicked()</signal>
   <receiver>CUpdateExtensionsBaseDialog</receiver>
   <slot>pushbuttonok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushbuttonreset_clicked()</slot>
  <slot>pushbuttoncancel_clicked()</slot>
  <slot>pushbuttonapply_clicked()</slot>
  <slot>pushbuttonok_clicked()</slot>
 </slots>
</ui>
